x-pack/plugins/cloud_security_posture/server/lib/check_for_findings.ts

```diff
@@ -6,15 +6,16 @@
  */
 
 import { ElasticsearchClient, type Logger } from '@kbn/core/server';
-import { LATEST_FINDINGS_INDEX_DEFAULT_NS } from '../../common/constants';
+import { LATEST_FINDINGS_INDEX_DEFAULT_NS, FINDINGS_INDEX_PATTERN } from '../../common/constants';
 
-export const isLatestFindingsIndexExists = async (
+export const checkForFindings = async (
```

Comment:
using `is` in names in order to indicate a boolean value is a naming convention best practice, I would prefer if you could keep the `is` in the name

maybe we can try `isLatestFindingsIndexPopulated` ?

BTW, since this function is very important, it may be worth adding a comment explaining why it only checks for the `latest-findings` index